packages/unit-vitest/src/helpers/install-quasar-plugin.ts

```diff
@@ -0,0 +1,23 @@
+import { config } from '@vue/test-utils';
+import { cloneDeep } from 'lodash-es';
+import { Quasar, QuasarPluginOptions } from 'quasar';
+import { beforeAll, afterAll } from 'vitest';
+
+export function installQuasar(options?: Partial<QuasarPluginOptions>) {
+  const globalConfigBackup = cloneDeep(config.global);
+
+  beforeAll(() => {
+    config.global.plugins.unshift([Quasar, options]);
+    config.global.provide._q_l_ = {
```

Comment:
@Evertvdw are these always needed? I think they're only needed when testing a QPage/QLayout scenario, and we should probably move them under a `qLayoutInjections` helper as we did for Jest AE

Reply:
There's also `_q_pc_` to manage AFAIK

Reply:
You may be right, I can't think of a scenario causing problems if we provide it, right now

`_q_pc_ ` should be the injection key for q-page-container component stuff, as `_q_l_` is for q-layout